### PR TITLE
Python 2/3 compatibility and Django 1.10 compatibility

### PR DESCRIPTION
**Pull request by justinoue**
Request to merge my changes for python 3 compatibility using six's urllib, and @lseelenbinder's changes for Django 1.10 compatibility.

### CODE REVIEW

**Comment by lseelenbinder** [comment]
@justinoue Thanks for pulling these in!

**Comment by kyrylo** [comment]
Could you please squash some of your commits? That is, I can see that https://github.com/airbrake/airbrake-django/pull/11/commits/66bfff863f0783d59ec2f8fa8ef7fd8c3cf43a61 was replaced by https://github.com/airbrake/airbrake-django/pull/11/commits/a918aa5be14ea7652b12f412d32134c8283f310a, etc. This will help me to review the PR better.

**Comment by phumpal** [comment]
@justinoue @lseelenbinder ping.

Please see @kyrylo's [comment](https://github.com/airbrake/airbrake-django/pull/11#issuecomment-250438273).

**Comment by justinoue** [comment]
There's so few line changes the final diff is very small. Considering moving to bugsnag.

**Comment by kyrylo** [comment]
We try to maintain our Git history properly, but yeah, the final diff is indeed small. It’s a bit hard to review this because there are no tests, so may I ask you to squash it and show an example of the error you stumbled upon?

**Comment by phumpal** [comment]
@kyrylo I don't think there are outstanding issues with the Django notifier except the current notifier does not properly support python3.x.

Adding [six](https://pypi.python.org/pypi/six) provides the utility functions enabling support for code written in python2.x (>= 2.6) and python3.x. (e.g. the `urllib2` module has been split across multiple modules in python3.x).

@justinoue getting this squashed would definitely help keep our history clean. I think two commits, one for python 2/3 compatibility and one for django middleware, would be ideal.

Can you do please squash and we will get this merged? If you have time it would also be nice to see the error you experienced which resulted in this PR.

Thanks @justinoue @lseelenbinder

**Comment by kyrylo** [comment]
Fixes:

```
  File "/usr/local/lib/python3.6/site-packages/airbrake/utils/client.py", line 4, in <module>
    import urllib2
ModuleNotFoundError: No module named 'urllib2'
```

I checked that this code works both on python3/django-1.10 and python2/django-1.9.

File: airbrake/utils/client.py [hunk 1]
```diff
@@ -22,10 +22,10 @@ class Client(object):
     @property
     def url(self):
         scheme = 'http'
-        if self.settings['USE_SSL']:
+        if self.settings.get('USE_SSL', False):
```
Comment (kyrylo) [inline airbrake/utils/client.py]: We use SSL by default now, therefore this should be `True`.

**Comment by kyrylo** [comment]
@justinoue I'm happy to merge this, but could you please fix https://github.com/airbrake/airbrake-django/pull/11#discussion_r95370400?

**Comment by kyrylo** [comment]
@justinoue @lseelenbinder thanks for your contribution, I made the edit myself, hence this is replaced by https://github.com/airbrake/airbrake-django/issues/14

**Pull request closed without merging**